src/ports/analyticsDayData/component.ts

```diff
@@ -27,10 +39,10 @@ export function createAnalyticsDayDataComponent(options: {
         : getAnalyticsDayDataQuery(filters)
 
     const { analytics: fragments } = await subgraph.query<{
-      analytics: AnalyticsDataFragment[]
+      analytics: any[]
```

Comment:
Using any because the return data from the rentals graph is different than the one in the collections/marketplace graph

Reply:
Isn't there any more generic type we can write to avoid the `any`?

Reply:
Using generics now